tsconfig.json

```
@@ -1,13 +1,13 @@
 {
   "compilerOptions": {
-    "module": "Node16",
-    "moduleResolution":"Node16",
+    "module": "ES6",
+    "target": "ES5",
```

```suggestion
    "target": "ES6",
```
IIRC, ESM was introduced in ES6, so i think the compilation target should be no lower than ES6.